src/kensetsu-vaults.md

```diff
@@ -0,0 +1,106 @@
+# Kensetsu Platform
+
+## Theory
+
+### Overall description of the platform
+
+Kensetsu is a lending platform with an overcollateralised stablecoin pegged to the USD. It is a MakerDAO analogue in the SORA ecosystem.
+The native token for the Kensetsu platform is the KEN token, which is used for farming and protocol rewards.
+
+### Key Features - KUSD borrowing (available collateral, parameters). Fee Calculation Process
+
+A Vault is the primary entity which operates on the Kensetsu platform. The Vault has the following parameters;
+
+- **Collateral token** — A token that operates as collateral in borrowing operations. The Kensetsu platform operates with six key tokens: XOR, VAL, TBCD, PSWAP, ETH, and DAI.
+- **Debt in borrowed token** - This shows the amount of debt that the user borrowed. The Kensetsu platform allows borrowing of KUSD tokens pegged to USD.
+- **Available to borrow** - Indicates how much a user can borrow until their vault becomes liquidated.
+- **LTV (loan-to-value)** - The debt ratio to the collateral value in your vault, calculated as a percentage.
+- **Stability Fee** - An annual fee on the amount drawn from your vault, calculated as a percentage.
+- **Liquidation penalty** - a fee applied if the value of your collateral falls below a certain threshold, prompting the sale of assets to cover debt. This penalty increases debt.
+
+### Key actions available for Users:
+
+- **Create a vault** — This operation allows users to create a vault in the Kensetsu platform. Users must deposit some collateral tokens to create a vault. Users can not create empty vaults with collateral tokens, but they can create a vault without borrowing tokens.
```

Comment:
It is possible to deposit and borrow during CDP creation in one tx for better UX.